src/dmd/backend/dwarfdbginf.d

```diff
@@ -3194,11 +3162,9 @@ static if (1)
              */
             abbrev_table = AApair.create(debug_abbrev.buf.bufptr);
 
-        /* Write new entry into debug_abbrev.buf
```

Comment:
Remove all changes below this to a separate PR

Reply:
Ah, thanks, forgot to split that.

Reply:
Done